hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -148,6 +150,11 @@ public class DBScanner implements Callable<Void>, SubcommandWithParent {
       defaultValue = "10")
   private int threadCount;
 
+  @CommandLine.Option(names = {"--pre-file-records"},
+      description = "The number of print records per file.",
+      defaultValue = "-1")
+  private long preFileRecords;
```

Comment:
Nit: suggest `--max-records-per-file`

Reply:
Thanks for your comment and review, @xichen01 .
I will update soon.

Reply:
Please also rename `preFileRecords` to `recordsPerFile`.

(`pre` means "before")